pkg/controller/registry/resolver/resolver.go

```diff
@@ -55,7 +55,7 @@ func (w *debugWriter) Write(b []byte) (int, error) {
 	return n, nil
 }
 
-func (r *Resolver) Resolve(namespaces []string, subs []*v1alpha1.Subscription) ([]*cache.Entry, error) {
+func (r *Resolver) Resolve(namespaces []string, subs []*v1alpha1.Subscription, existingEntryPredicates ...cache.Predicate) ([]*cache.Entry, error) {
```

Comment:
As we talked about this in the meeting already but I wanted to point it out here for another folks to be aware of: this change is basically an undo of what Ben Luddy did. Not a big deal but something to be aware of.

Reply:
Addressed in https://github.com/operator-framework/operator-lifecycle-manager/pull/2743